lua/autorun/server/bounties.lua

```diff
@@ -44,10 +44,10 @@ end
 
 RegisterChatCommand({'bounty','setbounty'},function(ply,arg)
 	local t = string.Explode(" ",arg)
-	local to,c = PlyCount(string.Implode(table.remove(t)))
+	local p = tonumber(table.remove(t))
+	local to,c = PlyCount(string.Implode(t))
```

Comment:
to will be nil if there is not a matching player. you can't call :IsPlayer on nil. Just change the if statement to if c==1 and assume to is a player